src/python/turicreate/toolkits/image_classifier/_annotate.py

```diff
@@ -119,10 +119,6 @@ def annotate(data, image_column=None, annotation_column='annotations'):
                             [image_column],
                             annotation_column
                         )
-    if __platform == "linux" or __platform == "linux2":
-        model = _image_feature_extractor._create_feature_extractor("squeezenet_v1.1")
-        feature_sframe = model.extract_features(data, image_column, verbose=False, batch_size=64)
-        annotation_window.add_image_features(feature_sframe)
```

Comment:
Note: since this doesn't appear to be implemented on the server side yet, I opened #2716 to add it.